packages/vaadin-lumo-styles/src/components/tabs.css

```diff
@@ -0,0 +1,285 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+:host {
+  display: flex;
+  align-items: center;
+  -webkit-tap-highlight-color: transparent;
+}
+
+:host([hidden]) {
+  display: none !important;
+}
+
+:host([orientation='vertical']) {
+  display: block;
+  box-shadow: -1px 0 0 0 var(--lumo-contrast-10pct);
+}
+
+:host([orientation='vertical']) [part='tabs'] {
+  height: 100%;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+  width: 100%;
+  margin: 0.5rem 0;
+}
+
+:host(:not([orientation='vertical'])) {
+  box-shadow: inset 0 -1px 0 0 var(--lumo-contrast-10pct);
+  position: relative;
+  min-height: var(--lumo-size-l);
+}
+
+:host([orientation='horizontal']) [part='tabs'] {
+  flex-grow: 1;
+  display: flex;
+  align-self: stretch;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  margin: 0 0.75rem;
+}
+
+:host([orientation='horizontal']) [part='tabs'] ::slotted(vaadin-tab:not([theme~='icon-on-top'])) {
+  justify-content: center;
+}
+
+/*
+  This seems more future-proof than `overflow: -moz-scrollbars-none` which is marked obsolete
+  and is no longer guaranteed to work:
+  https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#Mozilla_Extensions
+*/
+@-moz-document url-prefix() {
+  :host([orientation='horizontal']) [part='tabs'] {
+    overflow: hidden;
+  }
+}
+
+:host([orientation='horizontal']) [part='tabs']::-webkit-scrollbar {
+  display: none;
+}
+
+[part='back-button'],
+[part='forward-button'] {
+  pointer-events: none;
+  opacity: 0;
+  cursor: default;
+  position: absolute;
+  z-index: 1;
+  font-family: lumo-icons;
+  color: var(--lumo-tertiary-text-color);
+  font-size: var(--lumo-icon-size-m);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 1.5em;
+  height: 100%;
+  transition: 0.2s opacity;
+  top: 0;
+}
+
+[part='forward-button']:hover,
+[part='back-button']:hover {
+  color: inherit;
+}
+
+:host([overflow~='start']) [part='back-button'],
+:host([overflow~='end']) [part='forward-button'] {
+  pointer-events: auto;
+  opacity: 1;
+}
+
+[part='back-button']::after {
+  content: var(--lumo-icons-angle-left);
+}
+
+[part='forward-button']::after {
+  content: var(--lumo-icons-angle-right);
+}
+
+:host([orientation='vertical']) [part='back-button'],
+:host([orientation='vertical']) [part='forward-button'] {
+  display: none;
+}
+
+/* Tabs overflow */
+
+[part='tabs'] {
+  --_lumo-tabs-overflow-mask-image: none;
+  -webkit-mask-image: var(--_lumo-tabs-overflow-mask-image);
```

Comment:
BTW, just realised that we can probably also drop `-webkit-mask-image` as the unprefixed property is [supported](https://caniuse.com/?search=-webkit-mask-image) in newer Safari versions.